app/views/Mails/acceptrefuse/sendProposalApproved.scala.html

```diff
@@ -2,59 +2,57 @@
-    <hr>
+    <p>L'équipe @Messages("longYearlyName")</p>
+    <br>
 }
+<hr>
 <h3>Welcome to @Messages("longYearlyName")</h3>
 <p>Hi,</p>
-<p>We're glad to inform you that your proposal for @Messages("longYearlyName") has been <strong>accepted</strong></p>
-<p>Proposal <strong>@proposal.title</strong>, of type [<strong>@Messages(proposal.talkType.id)</strong>].</p>
-<hr>
-<h3>What's next?</h3>
-<p>You must now validate or decline the general terms and conditions, and confirm that you are still available to do this presentation :</p>
-<a href="@ConferenceDescriptor.current().conferenceUrls.cfpURL@routes.ApproveOrRefuse.showAcceptTerms()">@ConferenceDescriptor.current().conferenceUrls.cfpURL@routes.ApproveOrRefuse.showAcceptTerms()</a>
+<p>We're glad to inform you that your proposal <strong>@proposal.title</strong> (<strong>@Messages(proposal.talkType.id)</strong>) has been <strong>accepted</strong> for @Messages("longYearlyName")!</p>
+<br>
+<h3>How to validate or not your participation?</h3>
+<p>Note that you only have a few days to accept the terms and conditions and confirm your presence at @Messages("longYearlyName"). </p>
+<p><strong>Please validate now by following this link in your browser:</strong> <a href="@ConferenceDescriptor.current().conferenceUrls.cfpURL@routes.ApproveOrRefuse.showAcceptTerms()">@ConferenceDescriptor.current().conferenceUrls.cfpURL@routes.ApproveOrRefuse.showAcceptTerms()</a></p>
+<p>You will then have to accept or refuse your presentation(s).</p>
+<br>
+<h3>A doubt? A question?</h3>
+<p>If you can’t confirm quickly, if you have any questions, please come back to us as soon as possible.</p>
+<p><strong>Do not reply to this email</strong>, but use the interface to exchange with the committee:  <a href="@ConferenceDescriptor.current().conferenceUrls.cfpURL@routes.CallForPaper.showCommentForProposal(proposal.id)">@ConferenceDescriptor.current().conferenceUrls.cfpURL@routes.CallForPaper.showCommentForProposal(proposal.id)</a></p>
+<br>
+<br>
 
-    @if(!ConferenceDescriptor.ConferenceProposalConfigurations.doesItGivesSpeakerFreeEntrance(proposal.talkType)) {
-    <p>This type of talk does not grant a conference pass, so you will need to register as an attendee in order to access the conference area.
-    Registration details <a href="https://reg.devoxx.fr/speakers/welcome/@proposal.id/@email">register Devoxx FR 2019</a></p>
+@if(!ConferenceDescriptor.ConferenceProposalConfigurations.doesItGivesSpeakerFreeEntrance(proposal.talkType)) {
+    <p><strong>Attention</strong>, this presentation format for @Messages("longYearlyName") <strong>does not give a speaker badge</strong>. You must buy a seat at the conference to participate, if you do not have a ticket.</p>
+    <p>The conference being full, you can still register via the following unique link (and linked to your proposal): <a href="https://reg.devoxx.fr/speakers/welcome/@proposal.id/@email">https://reg.devoxx.fr/speakers/welcome/@proposal.id/@email</a></p>
```

Comment:
```suggestion
@if(!ConferenceDescriptor.ConferenceProposalConfigurations.doesItGivesSpeakerFreeEntrance(proposal.talkType)) {
    <p><strong>Please note that</strong>, this presentation format for @Messages("longYearlyName") <strong>does not provide a speaker badge</strong>. You will need to purchase a seat at the conference to attend, if you do not have a ticket.</p>
    <p>Since the conference is full, you can still register using the following unique link (linked to your proposal): <a href="https://reg.devoxx.fr/speakers/welcome/@proposal.id/@email">https://reg.devoxx.fr/speakers/welcome/@proposal.id/@email</a></p>
```